src/Notes/CouponPageMoved.php

```diff
@@ -33,6 +33,7 @@ public function init() {
 
 		add_action( 'admin_init', [ $this, 'possibly_add_note' ] );
 		add_action( 'admin_init', [ $this, 'redirect_to_coupons' ] );
+		add_action( 'woocommerce_admin_newly_installed', [ $this, 'disable_legacy_menu_for_new_install' ] );
```

Comment:
💅 

My 2 cents:

`add_action( 'woocommerce_admin_newly_installed', [ $this, 'display_legacy_menu' ] );`

Also works because false is the default param, no need for `disable_legacy_menu_for_new_install`. 

However, I would change the name of `display_legacy_menu` to `set_display_legacy_menu`

Reply:
That is what I added initially, however `add_action` needs a public function, `display_legacy_menu` is a protected function.